src/Basement/3DBasementHT.f90

```diff
@@ -2589,6 +2590,12 @@ SUBROUTINE BasementSimulator(RUNID,NMAT,CVG,XDIM,YDIM,ZDIM,TG)
 !***  Echo input data
       CALL PrelimOutput(ACEIL,AFLOOR,ARIM,ASILL,AWALL,PERIM,RUNID,TDBH,TDBC)
 
+      CALL GET_ENVIRONMENT_VARIABLE("CI_BAIL_EARLY", status=CI_BAIL_EARLY_STATUS)
+      IF (CI_BAIL_EARLY_STATUS == 0) THEN
+        print *, 'Exiting early because envionment variable CI_BAIL_EARLY was found'
+        CALL EXIT(0)
+      END IF
```

Comment:
Add an environment variable "CI_BAIL_EARLY": if defined, exit after CalcTearth & PrelimOutput, which is enough to produce the crash. I don't want to add a Basement test that takes like 10 minutes, or worse like an hour (when depth is 10.3, it's looooong)